charm.go

```diff
@@ -38,3 +40,29 @@ func ReadCharm(path string) (charm Charm, err error) {
 	}
 	return charm, nil
 }
+
+// SeriesToUse takes a specified series and a list of series supported by a
+// charm and returns the series which is relevant.
+// If the specified series is empty, the any default series defined by the
```

Comment:
If you're saying it's written as intended, then I do not understand it. I would write it as:

    If the requested series is empty, the charm's default series (first item in the supported series list) is used.

Is that correct?